scenes/song_parser/song.gd

```diff
@@ -4,6 +4,8 @@ class_name Song
 var song_music_file: String
 var song_music_buffer: PackedByteArray
 
+# Generic Variant used by song_loader.gd to load the song.
+var soog_music_file_access = FileAccess
```

Comment:
I did try to set it incorrectly on purpose, but didn't test that with the web build. good to know. I will try to fix it. The fetch should be ignored if the file is not found.

I will add a retry popup  in case it fails to allow the continuation in offline mode as well.